src/AutoRest.CSharp/Common/Input/Inputs.cs

```diff
@@ -161,7 +161,9 @@ internal record InputDictionaryType(string Name, InputType KeyType, InputType Va
 
     internal record InputModelProperty(string Name, string? SerializedName, string Description, InputType Type, bool IsRequired, bool IsReadOnly, bool IsDiscriminator) { }
 
-    internal record InputConstant(object? Value, InputType Type);
+    internal record InputConstant(object? Value, InputType Type) {
```

Comment:
Do we need `Value` to be nullable?

Reply:
value should not be null as constant. I will update it.